Mathlib/MeasureTheory/Function/AEEqFun.lean

```diff
@@ -121,18 +121,27 @@ def mk {β : Type*} [TopologicalSpace β] (f : α → β) (hf : AEStronglyMeasur
   Quotient.mk'' ⟨f, hf⟩
 #align measure_theory.ae_eq_fun.mk MeasureTheory.AEEqFun.mk
 
+/-- Auxiliary condition used in the cast function. -/
+def cast.Cond (f : α →ₘ[μ] β) : Prop :=
+  ∃ (g : α → β) (hg : StronglyMeasurable g),
+    f = mk g hg.aestronglyMeasurable ∧ g ∈ range (Function.const α)
```

Comment:
Why not use `∃ (b : β), f = mk (Function.const α b) ...` ?

Reply:
I could do that, originally I was planning to replace `g ∈ range (Function.const α)` with `g ∈ range (Function.const α) \or Continuous g`.

Reply:
Please do that.